Microfrontends example encountering a build error when deploying

Using the [clone button](https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/solutions/microfrontends&project-name=microfrontends&repository-name=microfrontends) on the `microfrontends` example, I get a failed build deploying to Vercel

```
main:build: error - ESLint: Failed to load parser './parser.js' declared in '.eslintrc.cjs » eslint-config-acme » eslint-config-next/core-web-vitals » /vercel/path0/node_modules/eslint-config-next/index.js': Cannot find module 'next/dist/compiled/babel/eslint-parser' Require stack: - /vercel/path0/node_modules/eslint-config-next/parser.js - /vercel/path0/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
--
12:40:35.037 | docs:build: error - ESLint: Failed to load parser './parser.js' declared in '.eslintrc.cjs » eslint-config-acme » eslint-config-next/core-web-vitals » /vercel/path0/node_modules/eslint-config-next/index.js': Cannot find module 'next/dist/compiled/babel/eslint-parser' Require stack: - /vercel/path0/node_modules/eslint-config-next/parser.js - /vercel/path0/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
12:40:35.429 | @acme/storybook:build:
12:40:35.429 | @acme/storybook:build: attention => Storybook now collects completely anonymous telemetry regarding usage.
12:40:35.429 | @acme/storybook:build: This information is used to shape Storybook's roadmap and prioritize features.
12:40:35.429 | @acme/storybook:build: You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
12:40:35.429 | @acme/storybook:build: https://storybook.js.org/telemetry
12:40:35.430 | @acme/storybook:build:
12:40:36.265 | @acme/storybook:build: vite v2.9.15 building for production...
12:40:36.387 | @acme/storybook:build: transforming...
12:40:40.543 | docs:build: Failed to compile.
12:40:40.544 | docs:build:
12:40:40.545 | docs:build: ./pages/_app.tsx:1:31
12:40:40.545 | docs:build: Type error: Cannot find module 'next/app' or its corresponding type declarations.
12:40:40.545 | docs:build:
12:40:40.545 | docs:build: > 1 \| import type { AppProps } from 'next/app'
12:40:40.546 | docs:build:     \|                               ^
12:40:40.546 | docs:build:   2 \| import type { LayoutProps } from '@vercel/examples-ui/layout'
12:40:40.546 | docs:build:   3 \| import { getLayout } from '@vercel/examples-ui'
12:40:40.546 | docs:build:   4 \| import '@vercel/examples-ui/globals.css'
12:40:40.641 | docs:build: error Command failed with exit code 1.
12:40:40.642 | docs:build: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
12:40:40.675 | docs:build: ERROR: command finished with error: command (/vercel/path0/apps/docs) yarn run build exited (1)
12:40:40.695 | command (/vercel/path0/apps/docs) yarn run build exited (1)
12:40:40.698 |  
12:40:40.698 | Tasks:    1 successful, 2 total
12:40:40.698 | Cached:    0 cached, 2 total
12:40:40.698 | Time:    17.395s
12:40:40.699 |  
12:40:40.699 | ERROR  run failed: command  exited (1)
12:40:40.720 | error Command failed with exit code 1.
12:40:40.720 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
12:40:40.740 | Error: Command "yarn run build" exited with 1
```

--- a/solutions/microfrontends/README.md
+++ b/solutions/microfrontends/README.md
@@ -5,7 +5,7 @@ description: Microfrontends allow teams to work independently of each other by s
 framework: Next.js
 useCase: Documentation
 css: Tailwind
-deployUrl: https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/solutions/microfrontends&project-name=microfrontends&repository-name=microfrontends&root-directory=solutions/microfrontends/apps/main&install-command=pnpm%20install&build-command=cd%20..%2F..%20%26%26%20pnpm%20build%20--filter%3Dmain...&ignore-command=npx%20turbo-ignore
+deployUrl: https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/solutions/microfrontends&project-name=microfrontends&repository-name=microfrontends&root-directory=apps/main&install-command=pnpm%20install&build-command=pnpm%20build%20--filter%3Dmain...&ignore-command=npx%20turbo-ignore
 demoUrl: https://solutions-microfrontends.vercel.app
 relatedTemplates:
   - monorepo-nx
@@ -32,7 +32,7 @@ You can choose from one of the following two methods to use this repository:
 
 Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=vercel-examples):
 
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/solutions/microfrontends&project-name=microfrontends&repository-name=microfrontends&root-directory=solutions/microfrontends/apps/main&install-command=pnpm%20install&build-command=cd%20..%2F..%20%26%26%20pnpm%20build%20--filter%3Dmain...&ignore-command=npx%20turbo-ignore)
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/solutions/microfrontends&project-name=microfrontends&repository-name=microfrontends&root-directory=apps/main&install-command=pnpm%20install&build-command=pnpm%20build%20--filter%3Dmain...&ignore-command=npx%20turbo-ignore)
 
 ### Clone and Deploy
 

